src/components/prometheusmetrics.ts

```diff
@@ -380,11 +380,11 @@ export class PrometheusMetrics {
             // TODO: Ideally these metrics would be on a different port.
             // For now, leave this unauthenticated.
             checkToken: false,
-            handler: (_req: Request, res: Response) => {
+            handler: async (_req: Request, res: Response) => {
```

Comment:
This appears in #296 but I needed it to work for my encrypted bridge deployment #sorry

Reply:
No need to be sorry. Great work!